spec.md

```diff
@@ -580,6 +580,7 @@ GET /v2/<name>/referrers/<digest>?artifactType=application/vnd.example.sbom.v1
 Example response with filtering:
 
 ```json
+OCI-Filters-Applied: artifactType
```

Comment:
Should this mirror the annotation and be `OCI-Referrers-Filters-Applied`

Reply:
I don't think we need that much language. For the annotation, it made sense, because you have no context about where it's used when it's just an annotation. In this case it's obvious that the header is relevant to the response to the referrers API by virtue of it being a header on the response to the referrers API.

Reply:
Agreed and LGTM then.